Formula/g/grpc-swift.rb

```diff
@@ -1,10 +1,10 @@
 class GrpcSwift < Formula
   desc "Swift language implementation of gRPC"
   homepage "https://github.com/grpc/grpc-swift"
```

Comment:
Updated URLs but still need to rename. Repo is `grpc-swift-protobuf` though `protoc-gen-grpc-swift` is probably closer to other formulae (e.g. `protoc-gen-go-grpc`, `protoc-gen-grpc-web`, etc)